doc/Train a Deep Potential model to fit tensor like Dipole and Polarizability.md

```diff
@@ -0,0 +1,109 @@
+# Train a Deep Potential model to fit `tensor` like `Dipole` and `Polarizability`
+
+Unlike `energy` which is a scalar, one may want to fit some high dimensional physical quantity, like `dipole` (vector) and `polarizability` (matrix, shorted as `polar`). Deep Potential has provided different API to allow this. In this example we will show you how to train a model to fit them for a water system. A complete training input script of the examples can be found in 
+
+```bash
+$deepmd_source_dir/examples/water_tensor/dipole/dipole_input.json
+$deepmd_source_dir/examples/water_tensor/polar/polar_input.json
+```
+
+Relevant data are also provided in our examples. But note that **our examples are of very limited amount, and should not be used to train a productive model.**

```

Comment:
"Relevant data are also provided" -> "The training and validation data are also provided..." 

"our examples are of very limited amount" -> "The data provided along with the examples are of limited amount"